ppr-ui/src/store/state/state-model.ts

```diff
@@ -18,6 +18,29 @@ export const stateModel: StateModelIF = {
     certified: false,
     legalName: ''
   },
+  folioOrReferenceNumber: '',
```

Comment:
just moved here so its alphabetical to make the store a little easier to navigate now that we have quitte a few sections